lua/nvim-treesitter/install.lua

```diff
@@ -198,4 +198,28 @@ function M.install_parser(ft)
   run_install(cache_folder, package_path, ft, repository)
 end
 
+function M.checkhealth()
+  local health_ok = vim.fn['health#report_ok']
+  local health_info = vim.fn['health#report_info']
+  local health_warn = vim.fn['health#report_warn']
+  local health_error = vim.fn['health#report_error']
+
+  if fn.executable('git') == 0 then
+    health_error('`git` executable not found.', {
+      'Install it with your package manager.',
+      'Check that your `$PATH` is correctly set.'})
```

Comment:
`is set correctly` would be the correct typo if i'm not mistaken. I could be wrong.